src/Python/importsd.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+#
+import torch
+import leb128
+import numpy as np
+from collections import OrderedDict
+
+
+_DTYPE_SIZE_MAP = {
+    np.uint8: 1,
+    np.int8: 1,
+    np.int16: 2,
+    np.int32: 4,
+    np.int64: 8,
+    np.float16: 2,
+    np.float32: 4,
+    np.float64: 8,
+}
+
+
+def _get_elem_type(type_num: int):
+    if type_num == 0:
+        return np.uint8
+    elif type_num == 1:
+        return np.int8
+    elif type_num == 2:
+        return np.int16
+    elif type_num == 3:
+        return np.int32
+    elif type_num == 4:
+        return np.int64
+    elif type_num == 5:
+        return np.float16
+    elif type_num == 6:
+        return np.float32
+    elif type_num == 7:
+        return np.float64
+    elif type_num == 0:
+        # return torch.bool
```

Comment:
Do not know how to calculate the size of bool and bfloat16

Reply:
I'm confused -- 'type_num == 0' is your first case in this long if-then-else.